app/models/card.py

```diff
@@ -1 +1,26 @@
-from app import db
+
+from app import db
+
+class Card(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True, nullable=False)
+    message = db.Column(db.String, nullable=False)
+    likes_count = db.Column(db.Integer, default=0)
+    status = db.Column(db.Boolean, default=True)

```

Comment:
`status` isn't a very specific word, it's unclear what it represents without digging into how it gets used in the routes. I strongly suggest renaming this to `is_archived`, `marked_inactive`, or something along those lines that makes it clear to a reader what this boolean means for the card.